### PR TITLE
Define handleFormattedTextClick as an arrow function

### PR DESCRIPTION
**Pull request by nevyangelova**
https://mattermost.atlassian.net/browse/MM-23045

Fixes https://github.com/mattermost/mattermost-webapp/pull/4988#issuecomment-595347421 

this was introduced after merging https://github.com/mattermost/mattermost-webapp/pull/4402

### CODE REVIEW

File: components/post_view/message_attachments/message_attachment/message_attachment.jsx [hunk 1]
```diff
@@ -242,9 +242,7 @@ export default class MessageAttachment extends React.PureComponent {
         );
     };
 
-    handleFormattedTextClick(e) {
-        handleFormattedTextClick(e, this.props.currentRelativeTeamUrl);
-    }
+    handleFormattedTextClick = (e) => handleFormattedTextClick(e, this.props.currentRelativeTeamUrl);
```
Comment (Willyfrog) [inline components/post_view/message_attachments/message_attachment/message_attachment.jsx]: can we change it like the others into Util.handleFormattedTextClick for the imported one so it is easy to differentiate between them?
I know it was like that earlier, but leaving it a bit more readable would be great.

**Comment by jgilliam17** [comment]
Thank you @nevyangelova 
Alt+click on user posts works as expected, no error.
Following errors are related to alt+click on the system message at the top of the channel and the system message with (Only visible to you) text. 

Note: Edge cases, as user may alt+click on sys message by accident. I checked and these errors are also present on the current release server, so I'm unsure if it should be fixed here, filed separately or needs further discussion. @sudheerDev, @Willyfrog what are your thoughts? Thanks.

Screen Shot 2020-03-07 at 8 50 25 AM
Screen Shot 2020-03-07 at 8 49 41 AM

**Comment by sudheerDev** [comment]
>  checked and these errors are also present on the current release server, so I'm unsure if it should be fixed here, filed separately

@jgilliam17 I think we should file them separately. The changes that caused this regression are only present in master. 

Just to be clear are you saying the above error is present in 5.21v?

**Comment by jgilliam17** [comment]
Yes, I can see this alt+click on system message error on 5.21 release server.

**Comment by sudheerDev** [comment]
@jgilliam17 Ya, I suggest we log it as a separate ticket and i just checked and it isn't a regression either.

**Pull request merged**